sqlstore/src/main/resources/db/migration_afterMigrate/05_trigger_sampleqc.sql

```diff
@@ -20,11 +21,12 @@ FOR EACH ROW
       CASE WHEN NEW.results <> OLD.results 
         THEN CONCAT('Updated ', (SELECT name FROM QCType WHERE qcTypeId = NEW.type), ' QC: ', OLD.results, ' → ', NEW.results, (SELECT units FROM QCType WHERE qcTypeId = NEW.type)) END);
       IF log_message IS NOT NULL AND log_message <> '' THEN
-        INSERT INTO SampleChangeLog(sampleId, columnsChanged, userId, message) VALUES (
+        INSERT INTO SampleChangeLog(sampleId, columnsChanged, userId, message, changeTime) VALUES (
           NEW.sample_sampleId,
           'QC',
           (SELECT lastModifier FROM Sample WHERE sampleId = NEW.sample_sampleId),
-          log_message);
+          log_message,
+          (SELECT lastModified FROM Sample WHERE sampleId = NEW.sample_sampleId));
```

Comment:
Could use INSERT INTO -> SELECT syntax to reduce subqueries on all these similar statements instead of finding `lastModifier` and `lastModified` separately